Azure/azure-cli
# [ACR] az acr import: Support no-wait

**Description**

Fixes https://github.com/Azure/azure-cli/issues/17851

**Testing Guide**
`az acr import -n MyRegistry --source sourceregistry.azurecr.io/sourcerepository:sourcetag --no-wait`


---

This checklist is used to make sure that common guidelines for a pull request are followed.
